src/components/CreateJobPage/CreateJobPage.js

```diff
@@ -45,7 +45,7 @@ const CreateJobPage = ({
       return setFunctions(groupedFunctions)
     })
 
-    if (functionsStore.templatesCatalog.length === 0) {
+    if (!functionsStore.templatesCatalog.length) {
```

Comment:
I actually prefer consider `.length` a number and compare it to `0` rather than considering it a "boolean" and using `!`.
Did you do it for the case when `functionsStore.templatesCatalog` doesn't even have a `length` property?